x-pack/plugins/ml/public/datavisualizer/datavisualizer.html

```diff
@@ -112,28 +156,49 @@ <h2 class="kuiSubTitle kuiVerticalRhythm">Metrics</h2>
 
       <div class="kuiPanel kuiVerticalRhythm datavisualizer-panel card-panel">
         <div class="euiText">
-          <h2 class="kuiSubTitle kuiVerticalRhythm">Fields</h2>
+          <h2
+            class="kuiSubTitle kuiVerticalRhythm"
+            i18n-id="xpack.ml.datavisualizer.fieldsTitle"
+            i18n-default-message="Fields"
+          ></h2>
         </div>
 
         <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--responsive euiFlexGroup--wrap">
           <div class="field-sort-controls euiFlexItem euiFlexItem--flexGrowZero">
             <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--responsive">
               <div ng-if="showAllFields === false" class="field-count-label euiFlexItem euiFlexItem--flexGrowZero">
-                <span class="field-emphasis">{{ fieldCards.length }}</span>
-                <ng-pluralize count="fieldCards.length" when="{'1':'field exists', 'other':'fields exist'}">fields exist</ng-pluralize> in documents
-                (<span class="field-emphasis">{{ totalNonMetricFieldCount }}</span> in total)
+                <span
+                  i18n-id="xpack.ml.datavisualizer.showMFieldDescription"  
+                  i18n-default-message="{wrappedFieldCardsCount} {fieldCardsCount, plural, one {field exists} other {fields exist}} in documents ({totalNonMetricFieldCount} in total)"
+                  i18n-values="{
+                      wrappedFieldCardsCount: '<span class=\'field-emphasis\'>' + fieldCards.length + '</span>',
```

Comment:
As above, the `<span>` tag should not be inside the `i18n-values` value

![image](https://user-images.githubusercontent.com/7405507/49738866-5d8be080-fc88-11e8-87ec-c92d69de39b4.png)

Reply:
With the last **master** <code>\<span\></code>  tag render correctly
![image](https://user-images.githubusercontent.com/25555662/50217948-560cbb80-039b-11e9-8800-5c21f684d627.png)